.appveyor/build_task.cmd

```diff
@@ -0,0 +1,73 @@
+@echo off
+setlocal enableextensions enabledelayedexpansion
+
+	cd /D %APPVEYOR_BUILD_FOLDER%
+	if %errorlevel% neq 0 exit /b 3
+
+	set STABILITY=staging
+	set DEPS_DIR=%PHP_BUILD_CACHE_BASE_DIR%\deps-%PHP_REL%-%PHP_SDK_VC%-%PHP_SDK_ARCH%
+	rem SDK is cached, deps info is cached as well
+	echo Updating dependencies in %DEPS_DIR%
+	cmd /c phpsdk_deps --update --no-backup --branch %PHP_REL% --stability %STABILITY% --deps %DEPS_DIR%
+	if %errorlevel% neq 0 exit /b 3
+
+	rem Something went wrong, most likely when concurrent builds were to fetch deps
+	rem updates. It might be, that some locking mechanism is needed.
+	if not exist "%DEPS_DIR%" (
+		cmd /c phpsdk_deps --update --force --no-backup --branch %PHP_REL% --stability %STABILITY% --deps %DEPS_DIR%
+	)
+	if %errorlevel% neq 0 exit /b 3
+
+	if "%ZTS_STATE%"=="enable" set ZTS_SHORT=ts
+	if "%ZTS_STATE%"=="disable" set ZTS_SHORT=nts
+	if "%ZTS_STATE%"=="enable" set ZTS_IN_FILENAME=
+	if "%ZTS_STATE%"=="disable" set ZTS_IN_FILENAME=-nts
+	if "%PHP_SDK_ARCH%"=="x86" set ARCH_IN_FILENAME=
+	if "%PHP_SDK_ARCH%"=="x64" set ARCH_IN_FILENAME=-x86_64
+
+	cd /d C:\projects\php-src
+
+	cmd /c buildconf.bat --force
+
+	if %errorlevel% neq 0 exit /b 3
+
+	cmd /c configure.bat --disable-all --with-mp=auto --enable-cli --%ZTS_STATE%-zts --enable-json --with-openssl --enable-mongodb=shared --enable-object-out-dir=%PHP_BUILD_OBJ_DIR% --with-config-file-scan-dir=%APPVEYOR_BUILD_FOLDER%\build\modules.d --with-prefix=%APPVEYOR_BUILD_FOLDER%\build --with-php-build=%DEPS_DIR%
```

Comment:
Other than the "mongodb" replacements here, was this basically copied from another project?

Reply:
Yes, copied from Xdebug, which was copied from something else.

We're still building php.exe too. I think that's fine for now, as "it works", and messing with this just takes too long for it to be worthwhile.

Reply:
Ok. IIRC, @weltling was talking about in #php.pecl:

```
<weltling> Derick, btw. perhaps you could check the phpize based approach https://github.com/krakjoe/apcu/blob/master/appveyor.yml 
<weltling> you can spare some time per single job by not building all the php, still need to fetch deps in your case
<weltling> see the build_script: section in the yml file
<weltling> for the tests - also of course php is needed, but in that case it fetches the precompiled version, too
<weltling> so except the extension itself, in most case nothing needs to be compiled
<weltling> with some exts i was helping out, the build time dropped reasonably
<weltling> here it fetches https://github.com/krakjoe/apcu/blob/master/appveyor.yml#L99
<weltling> basically, there it fetches the dev package, generates a job file and runs it, all packed into yml
<weltling> if there are no external deps, it's a way easier and faster approach, with deps might need some more lines
```

If you don't want to handle this in this PR, I would suggest creating a PHPC ticket so we can track this for later. Feel free to copy/paste that chat log into the ticket for future reference, too.